creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -0,0 +1,56 @@
+const promClient = require('prom-client')
+const _ = require('lodash')
+
+/**
+ * For explanation of Metrics, and instructions on how to add a new metric, please see `prometheusMonitoring/README.md`
+ */
+
+// We add a namespace prefix to differentiate internal metrics from those exported by different exporters from the same host
+const AudiusPrefix = 'audius_'
```

Comment:
Linking to this outdated discussion here: https://github.com/AudiusProject/audius-protocol/pull/3166#discussion_r896242940

Reply:
responded on thread